doc/sphinx/install.rst

```diff
@@ -0,0 +1,70 @@
+.. _install:
+
+Installation guide
+==================
+
+.. contents::
+
+
+C++
+---
+
+Pagmo is a header-only library which has the following third party dependencies:
+
+* `Boost <http://www.boost.org/>`_, headers only (needs the libraries only if you intend to compile the python bindings)
+* `Eigen <http://eigen.tuxfamily.org/index.php?title=Main_Page>`_, headers only (optional)
+
+
+After making sure the dependencies above are installed in your system and their headers visible to your compiler, you can download 
+pagmo source via the ``git`` command 
+
+.. code-block:: bash
+
+   git clone https://github.com/esa/pagmo2.git
+
+and configure your build using ``cmake``. When done, type (in your build directory):
+
+.. code-block:: bash
+
+   make install
+
+The headers will be installed in the CMAKE_INSTALL_PREFIX/include directory. To check that all went well compile the :ref:`quick-start example <getting_started_c++>`.
+
+-----------------------------------------------------------------------
+
+Python
+------
+The python module correponding to pagmo is called pygmo
+It can be installed either directly from ``conda`` or ``pip`` or by building the module from source.
+
+Installing with pip
+^^^^^^^^^^^^^^^^^^^
+The Python package pygmo (Python binding of the C++ code) can be installed using ``pip`` or ``conda``:
+
+.. code-block:: bash
+
+   pip install pygmo
+
+or
+
+.. code-block:: bash
+
+   conda install pygmo
```

Comment:
I am not 100% sure if this will work without adding the conda-forge channel first.